### PR TITLE
Make vsphere cluster configurable

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Fixes our e2e tests

### CODE REVIEW

File: examples/machine-vsphere.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ spec:
     cloudProviderSpec:
       templateVMName: ubuntu-template
       username: '<< VSPHERE_USERNAME >>'
-      vsphereURL: 'https://<< VSPHERE_ADDRESS >>'
+      vsphereURL: '<< VSPHERE_ADDRESS >>'
```
Comment (p0lyn0mial) [inline examples/machine-vsphere.yaml]: we could add a comment that would show an example value.

Comment (mrIncompetent) [reply examples/machine-vsphere.yaml]: I'll add an example

File: test/tools/integration/README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ creates a single-node-cluster (servertype: `cx11`) via `kubeadm` at
 
 ## Usage
 
-* `export HZ_TOKEN=<my_hetzner_cloud_token>`
+* `export HZ_E2E_TOKEN=<my_hetzner_cloud_token>`
```
Comment (p0lyn0mial) [inline test/tools/integration/README.md]: nice :)

**Comment by p0lyn0mial** [comment]
one nit and this is lgtm
@mrIncompetent thank you for fixing our e2e tests on vsphere !

**Pull request merged**